egs/timit/ASR/splits_dir/dev_samples.txt

```diff
@@ -0,0 +1,400 @@
+FADG0_SI1279 TEST/DR4/FADG0/SI1279.WAV
```

Comment:
Can this file be generated by some scripts? If so, we don't need to check it in.

Reply:
Em....about {train, dev, test} spliting files, I don't find there are some scripts to generate them. In kaldi, there are placed in [a list file](https://github.com/kaldi-asr/kaldi/blob/master/egs/timit/s5/conf/dev_spk.list). In speechbrain, there are placed in [timit_prepare.py](https://github.com/speechbrain/speechbrain/blob/develop/recipes/TIMIT/timit_prepare.py) which listing the speakers with a list. A option for us is to learn speechbrain. We can use a list to contain speakers' names in data preparing process. I will add it to Lhotse.